components/Buttons/src/MDCFloatingButton.h

```diff
@@ -96,6 +140,60 @@ typedef NS_ENUM(NSInteger, MDCFloatingButtonShape) {
 
 - (nullable instancetype)initWithCoder:(nonnull NSCoder *)aDecoder NS_DESIGNATED_INITIALIZER;
 
+- (void)setMinimumSize:(CGSize)size NS_UNAVAILABLE;
+
+/**
+ Sets the minimum size when the button has the specified @c shape @c mode.
+ Setting a size of @c CGSizeZero is equivalent to no minimum size.
+
+ @param minimumSize The new minimum size of the button.
+ @param shape The shape that the size constrains.
+ @param mode The mode that the size constrains.
+ */
+- (void)setMinimumSize:(CGSize)minimumSize
+              forShape:(MDCFloatingButtonShape)shape
+                inMode:(MDCFloatingButtonMode)mode UI_APPEARANCE_SELECTOR;
+
+- (void)setMaximumSize:(CGSize)maximumSize NS_UNAVAILABLE;
+
+/**
+ Sets the maximum size when the button has the specified @c shape and @c mode.
+ Setting a size of @c CGSizeZero is equivalent to no maximum size.
+
+ @param maximumSize The new maximum size of the button.
+ @param shape The shape that the size constrains.
+ @param mode The mode that the size constrains.
+ */
+- (void)setMaximumSize:(CGSize)maximumSize
+              forShape:(MDCFloatingButtonShape)shape
+                inMode:(MDCFloatingButtonMode)mode UI_APPEARANCE_SELECTOR;
+
+- (void)setContentEdgeInsets:(UIEdgeInsets)contentEdgeInsets NS_UNAVAILABLE;
+
+/**
+ Sets the @c contentEdgeInsets value when the button has the specified @c shape and @c mode.
```

Comment:
We should elaborate how this is different from a UIButton's contentEdgeInsets. Something like:
This descendent class of UIButton applies the contentEdgeInsets values after we perform a custom layout that pins the image to the edge of the button and maintains a space between the label and image as defined by @c imageTitleSpacing.

Reply:
I believe it should behave just like UIButton does.  Basically, we perform the following for layout:
1. Start with the button bounds
2. Inset by `contentEdgeInsets`
3. Position the image along the appropriate edge
4. Compute the space remaining for the title
5. Position and size the title
6. Apply `imageEdgeInsets` and `titleEdgeInsets`

Reply:
ok sgtm